src/inbox/mod_inbox.erl

```diff
@@ -536,13 +536,13 @@ config_metrics(HostType) ->
 
 muclight_dep(List) ->
     case lists:member(muclight, List) of
-        true -> [{mod_muc_light, hard}];
+        true -> [{mod_muc_light, [], hard}];
```

Comment:
From what I understand - this place, and all these similar ones could be a `#{}` already?

Reply:
If a module still has its options in a list, it should be `[]`.